runtime/connectors/local-file/local-file_test.go

```diff
@@ -0,0 +1,49 @@
+package local_file_test
+
+import (
+	"context"
+	"github.com/rilldata/rill/runtime/connectors"
+	"github.com/rilldata/rill/runtime/connectors/sources"
+	"github.com/rilldata/rill/runtime/drivers"
+	test_utils "github.com/rilldata/rill/runtime/test-utils"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+var curPath, _ = os.Getwd()
+var TestFilesPath = filepath.Join(curPath, "/../../../web-local/test/data/")
+
+func TestLocalFileConnector(t *testing.T) {
+	duckdb, err := test_utils.GetDuckdbDriver("stage.db")
```

Comment:
Don't think the separate test utils are necessary here. Can just add  `:= drivers.Open("duckdb", "...")`. Also, why not just use an in-memory DB to avoid cleanup?

If you do need to create files, use [`t.TempDir()`](https://pkg.go.dev/testing#T.TempDir), then they will automatically get cleaned up

Reply:
Also, remember to `defer db.Close()` (if I recall correctly)

Reply:
In memory is not supported by go bindings. We need to fix that in there and use in memory db here. For now i will disable the test from running in CI.